mimic/test/test_util.py

```diff
@@ -83,3 +83,27 @@ def test_unicode_body(self):
         self.failureResultOf(
             request(self, Resource(), b"POST", b"", u"not bytes")
         )
+
+
+class TestRandomString(SynchronousTestCase):
+    """
+    Tests for random string generation.
+    """
+
+    def test_length(self):
+        """
+        The random string you generate should have the length you specify.
+        """
+        for l in range(100):
+            self.assertEqual(len(helper.random_string(l)), l)
+
+    def test_selectable(self):
+        """
+        When passing a custom selectable, the results should derive only from
+        the characters you provide.
+        """
+        desired_chars = "02468"
+        for iteration in xrange(100):
+            a_string = helper.random_string(1024, selectable=desired_chars)
+            for char in a_string:
+                self.assertFalse(char not in desired_chars)
```

Comment:
Non-blocking style nitpick:  the double negative may be a little harder to read than just a positive assertion :)